depop-app/src/components/ProductsContainer.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import {ProductCard} from "./ProductCard";
+import {useGetProducts} from "../hooks/useGetProducts";
+import "./ProductsContainer.css"
+
+export const ProductsContainer = () => {
+
+    const {products, loading} = useGetProducts('https://5c78274f6810ec00148d0ff1.mockapi.io/api/v1/products');
+
+    return (
+        <React.Fragment>
+            {!loading && products &&
+                <div className={"subTitle"}><b>{products.length} Results</b></div>
```

Comment:
Let's leave this out and leave it for them to implement (often interesting to see how they handle this behaviour once products are filtered)

Reply:
Sure.